pkg/oci/client/config.go

```diff
@@ -19,25 +19,31 @@ import (
 	"io"
 	"io/ioutil"
 
+	"github.com/golang/glog"
 	"gopkg.in/yaml.v2"
 )
 
 // AuthConfig holds the configuration required for communicating with the OCI
 // API.
 type AuthConfig struct {
-	TenancyOCID           string `yaml:"tenancy"`
-	UserOCID              string `yaml:"user"`
-	CompartmentOCID       string `yaml:"compartment"`
-	PrivateKey            string `yaml:"key"`
-	Fingerprint           string `yaml:"fingerprint"`
-	Region                string `yaml:"region"`
-	PrivateKeyPassphrase  string `yaml:"passphrase"`
-	UseInstancePrincipals bool   `yaml:"useInstancePrincipals"`
+	TenancyOCID string `yaml:"tenancy"`
+	UserOCID    string `yaml:"user"`
+	// CompartmentOCID is DEPRECIATED and should be set on the top level Config
```

Comment:
Typo Deprecated => "Depreciated means the monetary value of something has decreased over time."